AdobeStockImageAdminUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/EntityType.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\AdobeStockImageAdminUi\Model\SearchCriteria\CollectionProcessor\FilterProcessor;
```

Comment:
Please move the files to `MediaGalleryUi` module.

Reply:
I tried moving the file AdobeStockImageAdminUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/EntityType.php
to
stock-integration/MediaGalleryUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/
i cannot get to work

Reply:
I meant that all the implementation should be moved to MediaGalleryUi module, not just a single file (for sure, the namespaces should be changed accordingly).

Reply:
Hi sergii, done moving the implementation MediaGalleryUi module